lib/common/format.h

```diff
@@ -102,6 +102,30 @@ template<typename CHAR = char> class FormatValidator {
     String,  // char-literal-constant or Hollerith constant
   };
 
+  bool mayHaveLeadingInt(TokenKind desc) {
```

Comment:
This is basically a set membership test; consider using `EnumSet`.

Reply:
Yes, it is.  Doesn't EnumSet bring C++ library requirements with it?

Reply:
Whether a dependence on the binary C++ runtime support library is created probably depends on what parts of `EnumSet` one uses.  I think you have a chance to just depend on things that are implemented in the standard headers in this case.